### ISSUE TITLE
loss is inf

### ISSUE THREAD

**Issue by DestinyK** [thstkdgus35/EDSR-PyTorch#198]
//EDSR in the paper (x2)
//python main.py --model EDSR --scale 2 --save edsr_x2 --n_resblocks 32 --n_feats 256 //res_scale 0.1 --reset --ext sep
I trained EDSR with this sh. It is normal the first time. I break it with Ctrl-c and trained the second time with the same sh. L1 is inf as:
                  [Epoch 1]       Learning rate: 1.00e-4
                  [800/8000]      [L1: inf]       110.2+10.1s
                  [1600/8000]     [L1: inf]       110.4+7.4s
                  [2400/8000]     [L1: inf]       112.9+7.6s

**Comment by thstkdgus35** [comment]
Hello.

It may depend on your initialization. You can try various initialization methods, or different [random seeds](https://github.com/thstkdgus35/EDSR-PyTorch/blob/565d6feaf58616558fb01a0c65342c01a3b3a26f/src/option.py#L18-L19) to avoid the problem.

Thank you
